src/components/Newsletter/Newsletter.style.tsx

```diff
@@ -58,33 +58,35 @@ export const StyledNewsletter = styled.div`
         display: none;
       }
     }
+  }
+`;
 
-    button[type='submit'] {
-      cursor: pointer;
-      display: block;
-      appearance: none;
-      box-sizing: border-box;
-      height: 3.5rem;
-      flex-shrink: 0;
-      flex: 1;
-      outline: 0;
-      border-top-right-radius: 4px;
-      border-bottom-right-radius: 4px;
-      appearance: none;
-      font-size: 0.875rem;
-      padding: 0.25rem 1.25rem;
-      line-height: 1rem;
-      text-transform: uppercase;
-      transition: 0.3s ease background-color;
-      background-color: ${palette.info.main};
-      border: 1px solid ${palette.info.dark};
-      color: ${palette.white};
-      font-weight: 700;
-      font-family: 'Roboto', 'Helvetica', 'Arial', sans-serif;
+export const SubscribeButton = styled.button<{ confirmation: boolean }>`
```

Comment:
I made this its own styled component so I could pass the `confirmation` prop to it.  I assume that's the right thing to do?

Reply:
yes, this should do it!